content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,34 @@
+---
+title: OpenTelemetry Metrics Release Candidates
+date: 2022-05-18
+author: Morgan McLean
+draft: true # TODO: remove this line once your post is ready to be published
+---
+
+OpenTelemetry’s metrics capabilities are now available as release candidates! This means that the specification, APIs, SDKs, and other components that author, capture, process, and otherwise interact with metrics now have the full set of OpenTelemetry metrics functionality and are ready for use. These release candidates will be promoted to general availability throughout the next few weeks.
+
+The 1.0 metrics release includes the following:
+
+- Metrics functionality included in the OpenTelemetry APIs for every GA-declared language, which provide language-specific interfaces that can create and manipulate metrics, and associate metadata and attributes to each. These are useful for:
+    - Developers of shared libraries that are distributed to end-users, so that these end users can natively use OpenTelemetry to capture metrics from these. For example, gRPC uses these APIs to produce latency, throughput, and error rate metrics for each RPC method on a given service.
+    - People who develop and maintain web services or client applications, so that they can produce custom metrics or interact with existing metrics. For example, an e-commerce company could use the API to track how many purchases are made over time.
```

Comment:
```suggestion
    - People who develop and maintain web services or client applications, so that they can produce custom metrics or interact with existing metrics. For example, an e-commerce company could use the API to track how many purchases are made over time.
    
```